site/en/docs/devtools/lighthouse/index.md

```diff
@@ -47,164 +57,111 @@ has 2 important functions:
 
 But first, you need to set up the site so that you can make changes to it later:
 
-1.  Go to `chrome://version` to check what version of Chrome you're using. This tutorial was made
-    with Chrome 68. If you're using an earlier or later version, some features may look different or
-    not be available. You should be able to complete the tutorial still, just keep in mind that your
-    UI may look different than the screenshots.
-2.  [Open the source code for the site][2]. This tab will be referred to as the _editor tab_.
-
-    {% Img src="image/admin/h2JSHSXIa8G3kOqRILbk.png", alt="The editor tab.", width="800", height="457" %}
-
-    **Figure 2**. The editor tab
-
-3.  Click **tony**. A menu appears.
-
-    {% Img src="image/admin/N9c7p2yrUhWgUobkDuVs.png", alt="The menu that appears after clicking 'tony'.", width="800", height="457" %}
-
-    **Figure 3**. The menu that appears after clicking **tony**
-
-4.  Click **Remix This**. The name of the project changes from **tony** to some randomly-generated
-    name. You now have your own editable copy of the code. Later on, you'll make changes to this
-    code.
-5.  Click **Show Live**. The demo opens in a new tab. This tab will be referred to as the _demo
-    tab_. It may take a while for the site to load.
+1. Open the [source of this demo website][2] and click **Remix** in the upper right corner. A new project opens in a tab. This tab will be referred to as the _editor tab_.
```

Comment:
You don't need to ask the user to select remix, in fact, u can add a remix link here, so when the user open, it's a new site right away.